pkg/plugin/watch.go

```diff
@@ -158,10 +160,7 @@ func (e *AutoscaleEnforcer) watchVMEvents(
 					return
 				}
 
-				updatedScalingBounds <- struct {
-					vm      *api.VmInfo
-					podName string
-				}{newInfo, newVM.Status.PodName}
+				updatedScalingBounds <- vmPodInfo{vm: newInfo, podName: newVM.Status.PodName}
```

Comment:
are we worried about backpressure here? should we use the context?

Reply:
Opened #160 for this